docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -604,6 +604,13 @@ run for any requester. If you specify an empty `allowed_requesters` list (i.e.
 `allow_for_git_tag`, or `git_tag_only` (if combined, the
 `allowed_requesters` will always take higher precedence).
 
+If `allowed_requesters` is specified and a conflicting project setting is also
+specified, `allowed_requesters` will take higher precedence. For example, if the
+project settings configure a [GitHub PR patch
+definition](Project-and-Distro-Settings.md#github-pull-request-testing) to run
+tasks A and B but task A has `allowed_requesters: ["commit"]`, then GitHub PR
+patches will only run task B.
+
 To cause a task to not run at all, set `disable: true`.
```

Comment:
This looks good but if you could do another drive by, could we add another header to this, or bullet point the sections or something? The above added stuff is long enouogh that I think this is going to get lost.

Reply:
I just moved the allowed requesters to its own section and moved this part up.